core-bundle/src/Framework/ContaoFramework.php

```diff
@@ -202,29 +198,6 @@ private function setDefaultLanguage(): void
         $GLOBALS['TL_LANGUAGE'] = $language;
     }
 
-    /**
-     * Redirects to the install tool if the installation is incomplete.
-     */
-    private function validateInstallation(): void
-    {
-        if (null === $this->request) {
-            return;
-        }
-
-        static $installRoutes = [
-            'contao_install',
-            'contao_install_redirect',
-        ];
-
-        if (\in_array($this->request->attributes->get('_route'), $installRoutes, true)) {
-            return;
-        }
-
-        if (!$this->getAdapter(Config::class)->isComplete()) {
```

Comment:
Should we also change `Config::isComplete()` to always return true now?

Reply:
Yes, and deprecate the method.

Reply:
https://github.com/contao/contao/pull/4938